selenium/test/simple.safari.spec.js

```diff
@@ -0,0 +1,42 @@
+const webdriver = require('selenium-webdriver');
+async function runTestWithCaps (capabilities) {
+  let driver = new webdriver.Builder()
+    .usingServer(`http://${process.env.SAUCE_USERNAME}:${process.env.SAUCE_ACCESS_KEY}@ondemand.saucelabs.com/wd/hub`)
```

Comment:
@nikolay-advolodkin this should use an endpoint such as \@ondemand.us-west-1.saucelabs.com/wd/hub. Might get some weird behaviour with this older style URL